curp/src/server/raw_curp/mod.rs

```diff
@@ -478,14 +478,13 @@ impl<C: 'static + Command, RC: RoleChange + 'static> RawCurp<C, RC> {
         let mut cst_w = self.cst.lock();
         debug!("{}'s vote is granted by server {}", self.id(), id);
 
-        cst_w.votes_received += 1;
+        let _ig = cst_w.votes_received.insert(id, vote_granted);
```

Comment:
The `vote_granted` is always true here, as it has been checked in L474. Please explain the usecase of the `votes_received` hasmap.

Reply:
Fixed, This is to prepare for the introduction of joint config in the future, there may be two configurations that use the same `votes_received` to check the vote result